embulk-jruby-strptime/src/main/java/org/embulk/spi/time/RubyDateParse.java

```diff
@@ -0,0 +1,283 @@
+package org.embulk.spi.time;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class RubyDateParse
+{
+    // Ported zones_source in ext/date/date_parse.c
```

Comment:
MRI 2.3.1 uses `zones_source` in ext/date/date_parse.rb. MRI 2.3.1 is used for JRuby 9.1.x. 
https://github.com/ruby/ruby/blob/394fa89c67722d35bdda89f10c7de5c304a5efb1/ext/date/date_parse.c